packages/zowe-explorer/scripts/downloadLocalizations.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+/**
+ * Downloads the latest translations submitted to Artifactory to
+ * https://zowe.jfrog.io/ui/repos/tree/General/libs-snapshot-local/org/zowe/vscode/zowe-explorer-g11n
+ * and copies them into the provides them into the top-level and l10n for building a localized extension.
+ */
+
+const path = require("path");
+const fs = require("fs");
+const process = require("process");
+const tmp = require("tmp");
```

Comment:
I know this is just a devDep, but why not use the `os.tmpdir()`? 😅

Reply:
See above the collapsed "github-advanced-security" message. The scan failed because of it.

Reply:
Thanks for pointing that out. 🙏 
Curious what checks make the package more secure than using `os.tmpdir` directly 😋